src/main/java/org/trigon/hardware/misc/leds/LEDStrip.java

```diff
@@ -0,0 +1,107 @@
+package org.trigon.hardware.misc.leds;
+
+import com.ctre.phoenix.led.LarsonAnimation;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.util.function.Supplier;
+
+public abstract class LEDStrip extends SubsystemBase {
+    public static LEDStrip[] LED_STRIPS = new LEDStrip[0];
+    final int indexOffset;
+    final boolean inverted;
+    final int numberOfLEDs;
+
+    public LEDStrip(boolean inverted, int numberOfLEDs, int indexOffset) {
+        this.indexOffset = indexOffset;
+        this.inverted = inverted;
+        this.numberOfLEDs = numberOfLEDs;
```

Comment:
These aren't in the same order as the params. Imo, the order should be: offset, number, inverted